api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -122,7 +122,9 @@ message StatusCodeFilter {
   ComparisonFilter comparison = 1 [(validate.rules).message = {required: true}];
 }
 
-// Filters on total request duration in milliseconds.
+// Filters based on the duration of the request or stream, in milliseconds.
+// For end of stream access logs, the total duration of the stream will be used.
```

Comment:
nit: Please add links to the "periodic access logs" [docs section](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/access_logging.html#periodic-access-logs).

Reply:
done